proxy/core/src/main/resources/META-INF/native-image/io.cloudstate/cloudstate-proxy-core/cloudstate-reflect-config.json.conf

```diff
@@ -0,0 +1,59 @@
+[
+  {
+    name: "io.cloudstate.proxy.autoscaler.KubernetesDeploymentScaler$Metadata"
+    allDeclaredFields: true
+    allPublicMethods: true
+    allPublicConstructors: true
+  }
+  {
+    name: "io.cloudstate.proxy.autoscaler.KubernetesDeploymentScaler$ScaleSpec"
+    allDeclaredFields: true
+    allPublicMethods: true
+    allPublicConstructors: true
+  }
+  {
+    name: "io.cloudstate.proxy.autoscaler.KubernetesDeploymentScaler$ScaleStatus"
+    allDeclaredFields: true
+    allPublicMethods: true
+    allPublicConstructors: true
+  }
+  {
+    name: "io.cloudstate.proxy.autoscaler.KubernetesDeploymentScaler$Scale"
+    allDeclaredFields: true
+    allPublicMethods: true
+    allPublicConstructors: true
+  }
+  {
+    name: "io.cloudstate.proxy.autoscaler.KubernetesDeploymentScaler$DeploymentCondition"
+    allDeclaredFields: true
+    allPublicMethods: true
+    allPublicConstructors: true
+  }
+  {
+    name: "io.cloudstate.proxy.autoscaler.KubernetesDeploymentScaler$DeploymentStatus"
+    allDeclaredFields: true
+    allPublicMethods: true
+    allPublicConstructors: true
+  }
+  {
+    name: "io.cloudstate.proxy.autoscaler.KubernetesDeploymentScaler$DeploymentSpec"
+    allDeclaredFields: true
+    allPublicMethods: true
+    allPublicConstructors: true
+  }
+  {
+    name: "io.cloudstate.proxy.autoscaler.KubernetesDeploymentScaler$Deployment"
+    allDeclaredFields: true
+    allPublicMethods: true
+    allPublicConstructors: true
+  }
+  {
+    name: "io.cloudstate.proxy.autoscaler.KubernetesDeploymentScaler$DeploymentList"
+    allDeclaredFields: true
+    allPublicMethods: true
+    allPublicConstructors: true
+  }
+
+  # Other
+  { name: com.google.protobuf.any.Any }
```

Comment:
I think it's mainly the inconsistency—putting names in a String literal seems like the thing to do, rather than wondering why some names are string literals and some not.

Reply:
Fine.